src/main/java/frc/robot/subsystems/PizzaBoxSubsystem/PizzaBoxSubsystem.java

```diff
@@ -19,17 +20,53 @@ public PizzaBoxSubsystem() {
 
   }
 
-  public Command reset_command(Servo s, Servo s2, TalonFX f) {
-    return this.runOnce(() -> PizzaBoxFuncs.reset(PBservo,PBFlapServo,m_PizzaBoxMotor));
+  public Command reset_command() {
+    return this.runOnce(() -> reset(PBservo,PBFlapServo,m_PizzaBoxMotor));
+  }
+
+  public Command spit_command(TalonFX motor) {
+    return this.runOnce(() -> motor.set(1));
   }
 
-  public Command spit_command(TalonFX m) {
-    return this.runOnce(() -> PizzaBoxFuncs.Spit(m));
+  public Command slurp_command(TalonFX motor) {
+    return this.runOnce(() -> motor.set(-1));
   }
 
-  public Command slurp_command(TalonFX m) {
-    return this.runOnce(() -> PizzaBoxFuncs.Slurp(m));
+  public Command stopMotor(TalonFX motor) {
+    return this.run(() -> motor.set(.00));
+  }
+
+  public static double motorSpeed(TalonFX motor) {
+    return motor.getRotorVelocity().getValue();
+  }
 
+  public Command stopFlap(Servo servo) {
+    return this.run(() -> servo.set(PizzaBoxConstants.STOP));
+  }
+public Command stopKicker(Servo servo) {
+    return this.run(() -> servo.set(PizzaBoxConstants.STOP));
+  }
+
+  public Command speedyArm_Command(TalonFX m) {
+    return this.run(() -> SpeedyArm(m));
+
+  }
+
+  public static String reset(Servo s, Servo s2, TalonFX f) {
+    s.set(PizzaBoxConstants.SERVO_NORM);
+    s2.set(PizzaBoxConstants.SERVO_NORM);
+    f.set(PizzaBoxConstants.STOP);
   
+    return "RESETING ANGLE (NO TOUCHY THE PIZZABOX)";
+
+    
+  }
+  public static void SpeedyArm(TalonFX m) {
+    if (ArmSubsystem.encoderGetAngle() < 99 && ArmSubsystem.encoderGetAngle() > 261) {
```

Comment:
Instead of getting the arm angle directly from the ArmSubsystem directly (and violating the command base programming pattern), a better approach is to take in a double supplier that supplies the arm angle.

https://docs.oracle.com/javase/8/docs/api/java/util/function/DoubleSupplier.html

Reply:
also, you flipped the less than and greater than.